src/kernel/src/sysctl/mod.rs

```diff
@@ -787,6 +811,37 @@ static KERN_ARANDOM: Oid = Oid {
     enabled: true,
 };
 
+static KERN_SDKVERSION: Oid = Oid {
+    parent: &KERN_CHILDREN,
+    link: Some(&KERN_CPUMODE), // TODO: Use a proper value.
+    number: Sysctl::KERN_SDKVERSION,
+    kind: Sysctl::CTLFLAG_RD
+        | Sysctl::CTLFLAG_MPSAFE
+        | Sysctl::CTLFLAG_CAPRD
+        | Sysctl::CTLTYPE_UINT,
+    arg1: None,
+    arg2: 0x09008031, // TODO: check what that means
+    name: "sdk_version",
+    handler: Some(Sysctl::kern_sdkversion),
```

Comment:
This handler might be one of the generic ones, please suggest which one if you have it reversed

Reply:
I think it's sysctl_handle_int

Reply:
Changed that